giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/core/service/processing/impl/CompletedStorageRequestProcessor.java

```diff
@@ -85,4 +87,14 @@ public String getProcessedTopic() {
     public Class<? extends ICompletedStorageRequest> getRequestClass() {
         return CompletedStorageRequest.class;
     }
+    
+    private void setOldFileVersionsIfReprocessing(IFile file) {
+        if (file.getStorageId() != null && !file.getStorageId().isEmpty()) {
+            if (file.getOldFileVersionsId() == null) {
+                file.setOldFileVersionsId(new ArrayList<>());
+            }
+            file.getOldFileVersionsId().add(file.getStorageId());
+            file.setOldFileVersionsId(file.getOldFileVersionsId());
```

Comment:
what is this line for?

Reply:
Before updating the storage id i will be adding the id to the oldFileVersionIds and updating the list